src/Http/Http.Abstractions/src/HttpProtocol.cs

```diff
@@ -18,13 +18,26 @@ public static class HttpProtocol
         // and allow us to optimize comparisons when these constants are used.
 
         // Please do NOT change these to 'const'
+
+        /// <summary>
+        /// Constant for HTTP protocol version 1.0.
```

Comment:
See earlier comment about constant on readonly fields.

Reply:
Fixed here and elsewhere.